packages/charts/src/components/legend/label.tsx

```diff
@@ -24,31 +24,45 @@ interface LabelProps {
 
 const isAppleDevice = typeof window !== 'undefined' && /Mac|iPhone|iPad/.test(window.navigator.userAgent);
 
-const modifierKey = isAppleDevice ? '⌘ (Command)' : 'Ctrl';
-const isolateSeriesMessage = 'isolate series';
-const showAllSeriesMessage = 'show all series';
-const showSeriesMessage = 'show series';
-const hideSeriesMessage = 'hide series';
+const modifierKey = isAppleDevice ? '⌘' : 'Ctrl';
+const showAllSeriesMessage = 'to show all';
+const showSeriesMessage = 'to show';
+const hideSeriesMessage = 'to hide';
 
 function getInteractivityTitle(isSeriesVisible: boolean, hiddenSeries: number, allSeries: number) {
   if (isSeriesVisible) {
     if (allSeries - hiddenSeries === 1) {
       return `
-Click: ${showAllSeriesMessage}
-${modifierKey} + click: ${hideSeriesMessage}`;
+Click ${showAllSeriesMessage}
+${modifierKey} + Click ${hideSeriesMessage}`;
     }
     if (hiddenSeries > 0) {
       return `
-Click: ${hideSeriesMessage}
-${modifierKey} + click: ${hideSeriesMessage}`;
+Click ${hideSeriesMessage}`;
     }
     return `
-Click: ${isolateSeriesMessage}
-${modifierKey} + click: ${hideSeriesMessage}`;
+Click ${showSeriesMessage}
+${modifierKey} + click ${hideSeriesMessage}`;
   }
   return `
-Click: ${showSeriesMessage}
-${modifierKey} + click: ${showSeriesMessage}`;
+Click ${showSeriesMessage}`;
+}
+
+function getInteractivityAriaLabel(isSeriesVisible: boolean, hiddenSeries: number, allSeries: number) {
+  if (isSeriesVisible) {
+    if (allSeries - hiddenSeries === 1) {
+      return `
+Click: ${showAllSeriesMessage}, ${modifierKey} + click: ${hideSeriesMessage}`;
```

Comment:
These don't matter as much but better to be consistent.

```suggestion
Click: ${showAllSeriesMessage}, ${modifierKey} + Click: ${hideSeriesMessage}`;
```